Failed to build hello world with CONFIG_POSIX_SYSCONF_IMPL_FULL=y

**Describe the bug**

Since cdc36d090c0c0abc1006f8c3f10100b6c54d385f (posix: sysconf: add a "small" implementation option), the hello world sample failed to build.

**To Reproduce**
```
west build -b nucleo_f429zi zephyr/samples/hello_world -- -DCONFIG_POSIX_SYSCONF=y -DCONFIG_POSIX_SYSCONF_IMPL_FULL=y
```

**Expected behavior**
Succeed to build the sample `hello_world`.

**Logs and console output**
```
❯ rm -rf build && west build -b nucleo_f429zi zephyr/samples/hello_world -- -DCONFIG_POSIX_SYSCONF=y -DCONFIG_POSIX_SYSCONF_IMPL_FULL=y         ─╯
-- west build: generating a build system
Loading Zephyr default modules (Zephyr base).
-- Application: /home/yashi/src/zephyr-project/zephyr/samples/hello_world
-- CMake version: 3.28.3
-- Found Python3: /home/yashi/src/zephyr-project/.venv/bin/python (found suitable version "3.11.8", minimum required is "3.8") found components: Interpreter 
-- Cache files will be written to: /home/yashi/.cache/zephyr
-- Zephyr version: 3.6.99 (/home/yashi/src/zephyr-project/zephyr)
-- Found west (found suitable version "1.2.0", minimum required is "0.14.0")
-- Board: nucleo_f429zi, identifier: stm32f429xx
-- ZEPHYR_TOOLCHAIN_VARIANT not set, trying to locate Zephyr SDK
-- Found host-tools: zephyr 0.16.5 (/home/yashi/zephyr-sdk-0.16.5)
-- Found toolchain: zephyr 0.16.5 (/home/yashi/zephyr-sdk-0.16.5)
-- Found Dtc: /usr/bin/dtc (found suitable version "1.7.0", minimum required is "1.4.6") 
-- Found BOARD.dts: /home/yashi/src/zephyr-project/zephyr/boards/st/nucleo_f429zi/nucleo_f429zi.dts
-- Generated zephyr.dts: /home/yashi/src/zephyr-project/build/zephyr/zephyr.dts
-- Generated devicetree_generated.h: /home/yashi/src/zephyr-project/build/zephyr/include/generated/devicetree_generated.h
-- Including generated dts.cmake file: /home/yashi/src/zephyr-project/build/zephyr/dts.cmake
Parsing /home/yashi/src/zephyr-project/zephyr/Kconfig
Loaded configuration '/home/yashi/src/zephyr-project/zephyr/boards/st/nucleo_f429zi/nucleo_f429zi_defconfig'
Merged configuration '/home/yashi/src/zephyr-project/zephyr/samples/hello_world/prj.conf'
Merged configuration '/home/yashi/src/zephyr-project/build/zephyr/misc/generated/extra_kconfig_options.conf'
Configuration saved to '/home/yashi/src/zephyr-project/build/zephyr/.config'
Kconfig header saved to '/home/yashi/src/zephyr-project/build/zephyr/include/generated/autoconf.h'
-- Found GnuLd: /home/yashi/zephyr-sdk-0.16.5/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/12.2.0/../../../../arm-zephyr-eabi/bin/ld.bfd (found version "2.38") 
-- The C compiler identification is GNU 12.2.0
-- The CXX compiler identification is GNU 12.2.0
-- The ASM compiler identification is GNU
-- Found assembler: /home/yashi/zephyr-sdk-0.16.5/arm-zephyr-eabi/bin/arm-zephyr-eabi-gcc
-- Using ccache: /usr/bin/ccache
CMake Warning at /home/yashi/src/zephyr-project/zephyr/CMakeLists.txt:864 (message):
  No SOURCES given to Zephyr library: lib__posix__shell

  Excluding target from build.


-- Configuring done (3.1s)
-- Generating done (0.0s)
-- Build files have been written to: /home/yashi/src/zephyr-project/build
-- west build: building application
[1/134] Preparing syscall dependency handling

[3/134] Generating include/generated/version.h
-- Zephyr version: 3.6.99 (/home/yashi/src/zephyr-project/zephyr), build: v3.6.0-851-gbe9a85bf6c5e
[100/134] Building C object zephyr/lib/posix/options/CMakeFiles/lib__posix__options.dir/sysconf.c.obj
FAILED: zephyr/lib/posix/options/CMakeFiles/lib__posix__options.dir/sysconf.c.obj 
ccache /home/yashi/zephyr-sdk-0.16.5/arm-zephyr-eabi/bin/arm-zephyr-eabi-gcc -DCORE_CM4 -DHSE_VALUE=8000000 -DKERNEL -DK_HEAP_MEM_POOL_SIZE=0 -DPICOLIBC_LONG_LONG_PRINTF_SCANF -DSTM32F429xx -DUSE_FULL_LL_DRIVER -DUSE_HAL_DRIVER -D__LINUX_ERRNO_EXTENSIONS__ -D__PROGRAM_START -D__ZEPHYR__=1 -I/home/yashi/src/zephyr-project/zephyr/kernel/include -I/home/yashi/src/zephyr-project/zephyr/arch/arm/include -I/home/yashi/src/zephyr-project/zephyr/include -I/home/yashi/src/zephyr-project/build/zephyr/include/generated -I/home/yashi/src/zephyr-project/zephyr/soc/st/stm32 -I/home/yashi/src/zephyr-project/zephyr/soc/st/stm32/common/. -I/home/yashi/src/zephyr-project/zephyr/drivers -I/home/yashi/src/zephyr-project/zephyr/soc/st/stm32/stm32f4x/. -I/home/yashi/src/zephyr-project/modules/hal/cmsis/CMSIS/Core/Include -I/home/yashi/src/zephyr-project/zephyr/modules/cmsis/. -I/home/yashi/src/zephyr-project/modules/hal/stm32/stm32cube/stm32f4xx/soc -I/home/yashi/src/zephyr-project/modules/hal/stm32/stm32cube/stm32f4xx/drivers/include -I/home/yashi/src/zephyr-project/modules/hal/stm32/stm32cube/stm32f4xx/drivers/include/Legacy -I/home/yashi/src/zephyr-project/modules/hal/stm32/stm32cube/common_ll/include -isystem /home/yashi/src/zephyr-project/zephyr/lib/libc/common/include -Wshadow -fno-strict-aliasing -Os -imacros /home/yashi/src/zephyr-project/build/zephyr/include/generated/autoconf.h -fno-printf-return-value -fno-common -g -gdwarf-4 -fdiagnostics-color=always -mcpu=cortex-m4 -mthumb -mabi=aapcs -mfp16-format=ieee -mtp=soft --sysroot=/home/yashi/zephyr-sdk-0.16.5/arm-zephyr-eabi/arm-zephyr-eabi -imacros /home/yashi/src/zephyr-project/zephyr/include/zephyr/toolchain/zephyr_stdint.h -Wall -Wformat -Wformat-security -Wno-format-zero-length -Wdouble-promotion -Wno-pointer-sign -Wpointer-arith -Wexpansion-to-defined -Wno-unused-but-set-variable -Werror=implicit-int -fno-pic -fno-pie -fno-asynchronous-unwind-tables -ftls-model=local-exec -fno-reorder-functions --param=min-pagesize=0 -fno-defer-pop -fmacro-prefix-map=/home/yashi/src/zephyr-project/zephyr/samples/hello_world=CMAKE_SOURCE_DIR -fmacro-prefix-map=/home/yashi/src/zephyr-project/zephyr=ZEPHYR_BASE -fmacro-prefix-map=/home/yashi/src/zephyr-project=WEST_TOPDIR -ffunction-sections -fdata-sections --specs=picolibc.specs -std=c99 -MD -MT zephyr/lib/posix/options/CMakeFiles/lib__posix__options.dir/sysconf.c.obj -MF zephyr/lib/posix/options/CMakeFiles/lib__posix__options.dir/sysconf.c.obj.d -o zephyr/lib/posix/options/CMakeFiles/lib__posix__options.dir/sysconf.c.obj -c /home/yashi/src/zephyr-project/zephyr/lib/posix/options/sysconf.c
In file included from /home/yashi/src/zephyr-project/zephyr/lib/posix/options/sysconf.c:7:
/home/yashi/src/zephyr-project/zephyr/lib/posix/options/sysconf.c: In function 'sysconf':
/home/yashi/src/zephyr-project/zephyr/include/zephyr/posix/unistd.h:137:45: error: 'CONFIG_MSG_COUNT_MAX' undeclared (first use in this function)
  137 | #define _POSIX_MQ_OPEN_MAX                  CONFIG_MSG_COUNT_MAX
      |                                             ^~~~~~~~~~~~~~~~~~~~
/home/yashi/src/zephyr-project/zephyr/include/zephyr/posix/unistd.h:192:39: note: in expansion of macro '_POSIX_MQ_OPEN_MAX'
  192 | #define MQ_OPEN_MAX                   _POSIX_MQ_OPEN_MAX
      |                                       ^~~~~~~~~~~~~~~~~~
/home/yashi/src/zephyr-project/zephyr/lib/posix/options/sysconf.c:227:24: note: in expansion of macro 'MQ_OPEN_MAX'
  227 |                 return MQ_OPEN_MAX;
      |                        ^~~~~~~~~~~
/home/yashi/src/zephyr-project/zephyr/include/zephyr/posix/unistd.h:137:45: note: each undeclared identifier is reported only once for each function it appears in
  137 | #define _POSIX_MQ_OPEN_MAX                  CONFIG_MSG_COUNT_MAX
      |                                             ^~~~~~~~~~~~~~~~~~~~
/home/yashi/src/zephyr-project/zephyr/include/zephyr/posix/unistd.h:192:39: note: in expansion of macro '_POSIX_MQ_OPEN_MAX'
  192 | #define MQ_OPEN_MAX                   _POSIX_MQ_OPEN_MAX
      |                                       ^~~~~~~~~~~~~~~~~~
/home/yashi/src/zephyr-project/zephyr/lib/posix/options/sysconf.c:227:24: note: in expansion of macro 'MQ_OPEN_MAX'
  227 |                 return MQ_OPEN_MAX;
      |                        ^~~~~~~~~~~
In file included from /home/yashi/src/zephyr-project/modules/hal/stm32/stm32cube/stm32f4xx/drivers/include/stm32f4xx_hal_def.h:30,
                 from /home/yashi/src/zephyr-project/modules/hal/stm32/stm32cube/stm32f4xx/drivers/include/stm32f4xx_hal_rcc.h:27,
                 from /home/yashi/src/zephyr-project/modules/hal/stm32/stm32cube/stm32f4xx/drivers/include/stm32f4xx_hal_conf.h:285,
                 from /home/yashi/src/zephyr-project/modules/hal/stm32/stm32cube/stm32f4xx/drivers/include/stm32f4xx_hal.h:29,
                 from /home/yashi/src/zephyr-project/modules/hal/stm32/stm32cube/stm32f4xx/soc/stm32f4xx.h:287,
                 from /home/yashi/src/zephyr-project/zephyr/soc/st/stm32/stm32f4x/./soc.h:23,
                 from /home/yashi/src/zephyr-project/zephyr/modules/cmsis/./cmsis_core_m.h:24,
                 from /home/yashi/src/zephyr-project/zephyr/modules/cmsis/./cmsis_core.h:10,
                 from /home/yashi/src/zephyr-project/zephyr/include/zephyr/arch/arm/mpu/arm_mpu_v7m.h:10,
                 from /home/yashi/src/zephyr-project/zephyr/include/zephyr/arch/arm/mpu/arm_mpu.h:14,
                 from /home/yashi/src/zephyr-project/zephyr/include/zephyr/arch/arm/arch.h:271,
                 from /home/yashi/src/zephyr-project/zephyr/include/zephyr/arch/cpu.h:19,
                 from /home/yashi/src/zephyr-project/zephyr/include/zephyr/kernel_includes.h:36,
                 from /home/yashi/src/zephyr-project/zephyr/include/zephyr/kernel.h:17,
                 from /home/yashi/src/zephyr-project/zephyr/include/zephyr/posix/posix_types.h:18,
                 from /home/yashi/src/zephyr-project/zephyr/include/zephyr/posix/unistd.h:9:
/home/yashi/src/zephyr-project/modules/hal/stm32/stm32cube/stm32f4xx/drivers/include/Legacy/stm32_hal_legacy.h:461:39: error: 'FLASH_PAGE_SIZE' undeclared (first use in this function); did you mean 'FLASH_CR_PSIZE'?
  461 | #define PAGESIZE                      FLASH_PAGE_SIZE
      |                                       ^~~~~~~~~~~~~~~
/home/yashi/src/zephyr-project/zephyr/lib/posix/options/sysconf.c:235:24: note: in expansion of macro 'PAGESIZE'
  235 |                 return PAGESIZE;
      |                        ^~~~~~~~
/home/yashi/src/zephyr-project/zephyr/include/zephyr/posix/unistd.h:219:39: error: 'CONFIG_MAX_TIMER_COUNT' undeclared (first use in this function)
  219 | #define TIMER_MAX                     CONFIG_MAX_TIMER_COUNT
      |                                       ^~~~~~~~~~~~~~~~~~~~~~
/home/yashi/src/zephyr-project/zephyr/lib/posix/options/sysconf.c:257:24: note: in expansion of macro 'TIMER_MAX'
  257 |                 return TIMER_MAX;
      |                        ^~~~~~~~~
[124/134] Building C object zephyr/kernel/CMakeFiles/kernel.dir/sched.c.obj
ninja: build stopped: subcommand failed.
FATAL ERROR: command exited with status 1: /usr/bin/cmake --build /home/yashi/src/zephyr-project/build
```

**Environment (please complete the following information):**
 - OS: Debian Sid
 - Toolchain: Zephyr SDK 0.16.5
 - Commit SHA: cdc36d090c0c0abc1006f8c3f10100b6c54d385f

it doesn't seems to be limited to stm32. `qemu_cortex_m3` failes as well.

```
❯ rm -rf build && west build -b qemu_cortex_m3 zephyr/samples/hello_world -- -DCONFIG_POSIX_SYSCONF=y -DCONFIG_POSIX_SYSCONF_IMPL_FULL=y        ─╯
-- west build: generating a build system
Loading Zephyr default modules (Zephyr base).
-- Application: /home/yashi/src/zephyr-project/zephyr/samples/hello_world
-- CMake version: 3.28.3
-- Found Python3: /home/yashi/src/zephyr-project/.venv/bin/python (found suitable version "3.11.8", minimum required is "3.8") found components: Interpreter 
-- Cache files will be written to: /home/yashi/.cache/zephyr
-- Zephyr version: 3.6.99 (/home/yashi/src/zephyr-project/zephyr)
-- Found west (found suitable version "1.2.0", minimum required is "0.14.0")
-- Board: qemu_cortex_m3, identifier: ti_lm3s6965
-- ZEPHYR_TOOLCHAIN_VARIANT not set, trying to locate Zephyr SDK
-- Found host-tools: zephyr 0.16.5 (/home/yashi/zephyr-sdk-0.16.5)
-- Found toolchain: zephyr 0.16.5 (/home/yashi/zephyr-sdk-0.16.5)
-- Found Dtc: /usr/bin/dtc (found suitable version "1.7.0", minimum required is "1.4.6") 
-- Found BOARD.dts: /home/yashi/src/zephyr-project/zephyr/boards/qemu/cortex_m3/qemu_cortex_m3.dts
-- Generated zephyr.dts: /home/yashi/src/zephyr-project/build/zephyr/zephyr.dts
-- Generated devicetree_generated.h: /home/yashi/src/zephyr-project/build/zephyr/include/generated/devicetree_generated.h
-- Including generated dts.cmake file: /home/yashi/src/zephyr-project/build/zephyr/dts.cmake
Parsing /home/yashi/src/zephyr-project/zephyr/Kconfig
Loaded configuration '/home/yashi/src/zephyr-project/zephyr/boards/qemu/cortex_m3/qemu_cortex_m3_defconfig'
Merged configuration '/home/yashi/src/zephyr-project/zephyr/samples/hello_world/prj.conf'
Merged configuration '/home/yashi/src/zephyr-project/build/zephyr/misc/generated/extra_kconfig_options.conf'
Configuration saved to '/home/yashi/src/zephyr-project/build/zephyr/.config'
Kconfig header saved to '/home/yashi/src/zephyr-project/build/zephyr/include/generated/autoconf.h'
-- Found GnuLd: /home/yashi/zephyr-sdk-0.16.5/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/12.2.0/../../../../arm-zephyr-eabi/bin/ld.bfd (found version "2.38") 
-- The C compiler identification is GNU 12.2.0
-- The CXX compiler identification is GNU 12.2.0
-- The ASM compiler identification is GNU
-- Found assembler: /home/yashi/zephyr-sdk-0.16.5/arm-zephyr-eabi/bin/arm-zephyr-eabi-gcc
-- Using ccache: /usr/bin/ccache
CMake Warning at /home/yashi/src/zephyr-project/zephyr/CMakeLists.txt:864 (message):
  No SOURCES given to Zephyr library: lib__posix__shell

  Excluding target from build.


-- Configuring done (2.8s)
-- Generating done (0.0s)
-- Build files have been written to: /home/yashi/src/zephyr-project/build
-- west build: building application
[1/110] Preparing syscall dependency handling

[3/110] Generating include/generated/version.h
-- Zephyr version: 3.6.99 (/home/yashi/src/zephyr-project/zephyr), build: v3.6.0-851-gbe9a85bf6c5e
[74/110] Building C object zephyr/lib/posix/options/CMakeFiles/lib__posix__options.dir/sysconf.c.obj
FAILED: zephyr/lib/posix/options/CMakeFiles/lib__posix__options.dir/sysconf.c.obj 
ccache /home/yashi/zephyr-sdk-0.16.5/arm-zephyr-eabi/bin/arm-zephyr-eabi-gcc -DKERNEL -DK_HEAP_MEM_POOL_SIZE=0 -DPICOLIBC_LONG_LONG_PRINTF_SCANF -D__LINUX_ERRNO_EXTENSIONS__ -D__PROGRAM_START -D__ZEPHYR__=1 -I/home/yashi/src/zephyr-project/zephyr/kernel/include -I/home/yashi/src/zephyr-project/zephyr/arch/arm/include -I/home/yashi/src/zephyr-project/zephyr/include -I/home/yashi/src/zephyr-project/build/zephyr/include/generated -I/home/yashi/src/zephyr-project/zephyr/soc/ti/lm3s6965 -I/home/yashi/src/zephyr-project/zephyr/soc/ti/lm3s6965/. -I/home/yashi/src/zephyr-project/modules/hal/cmsis/CMSIS/Core/Include -I/home/yashi/src/zephyr-project/zephyr/modules/cmsis/. -isystem /home/yashi/src/zephyr-project/zephyr/lib/libc/common/include -Wshadow -fno-strict-aliasing -Os -imacros /home/yashi/src/zephyr-project/build/zephyr/include/generated/autoconf.h -fno-printf-return-value -fno-common -g -gdwarf-4 -fdiagnostics-color=always -mcpu=cortex-m3 -mthumb -mabi=aapcs -mfp16-format=ieee -mtp=soft --sysroot=/home/yashi/zephyr-sdk-0.16.5/arm-zephyr-eabi/arm-zephyr-eabi -imacros /home/yashi/src/zephyr-project/zephyr/include/zephyr/toolchain/zephyr_stdint.h -Wall -Wformat -Wformat-security -Wno-format-zero-length -Wdouble-promotion -Wno-pointer-sign -Wpointer-arith -Wexpansion-to-defined -Wno-unused-but-set-variable -Werror=implicit-int -fno-pic -fno-pie -fno-asynchronous-unwind-tables -ftls-model=local-exec -fno-reorder-functions --param=min-pagesize=0 -fno-defer-pop -fmacro-prefix-map=/home/yashi/src/zephyr-project/zephyr/samples/hello_world=CMAKE_SOURCE_DIR -fmacro-prefix-map=/home/yashi/src/zephyr-project/zephyr=ZEPHYR_BASE -fmacro-prefix-map=/home/yashi/src/zephyr-project=WEST_TOPDIR -ffunction-sections -fdata-sections --specs=picolibc.specs -std=c99 -MD -MT zephyr/lib/posix/options/CMakeFiles/lib__posix__options.dir/sysconf.c.obj -MF zephyr/lib/posix/options/CMakeFiles/lib__posix__options.dir/sysconf.c.obj.d -o zephyr/lib/posix/options/CMakeFiles/lib__posix__options.dir/sysconf.c.obj -c /home/yashi/src/zephyr-project/zephyr/lib/posix/options/sysconf.c
In file included from /home/yashi/src/zephyr-project/zephyr/lib/posix/options/sysconf.c:7:
/home/yashi/src/zephyr-project/zephyr/lib/posix/options/sysconf.c: In function 'sysconf':
/home/yashi/src/zephyr-project/zephyr/include/zephyr/posix/unistd.h:137:45: error: 'CONFIG_MSG_COUNT_MAX' undeclared (first use in this function)
  137 | #define _POSIX_MQ_OPEN_MAX                  CONFIG_MSG_COUNT_MAX
      |                                             ^~~~~~~~~~~~~~~~~~~~
/home/yashi/src/zephyr-project/zephyr/include/zephyr/posix/unistd.h:192:39: note: in expansion of macro '_POSIX_MQ_OPEN_MAX'
  192 | #define MQ_OPEN_MAX                   _POSIX_MQ_OPEN_MAX
      |                                       ^~~~~~~~~~~~~~~~~~
/home/yashi/src/zephyr-project/zephyr/lib/posix/options/sysconf.c:227:24: note: in expansion of macro 'MQ_OPEN_MAX'
  227 |                 return MQ_OPEN_MAX;
      |                        ^~~~~~~~~~~
/home/yashi/src/zephyr-project/zephyr/include/zephyr/posix/unistd.h:137:45: note: each undeclared identifier is reported only once for each function it appears in
  137 | #define _POSIX_MQ_OPEN_MAX                  CONFIG_MSG_COUNT_MAX
      |                                             ^~~~~~~~~~~~~~~~~~~~
/home/yashi/src/zephyr-project/zephyr/include/zephyr/posix/unistd.h:192:39: note: in expansion of macro '_POSIX_MQ_OPEN_MAX'
  192 | #define MQ_OPEN_MAX                   _POSIX_MQ_OPEN_MAX
      |                                       ^~~~~~~~~~~~~~~~~~
/home/yashi/src/zephyr-project/zephyr/lib/posix/options/sysconf.c:227:24: note: in expansion of macro 'MQ_OPEN_MAX'
  227 |                 return MQ_OPEN_MAX;
      |                        ^~~~~~~~~~~
/home/yashi/src/zephyr-project/zephyr/include/zephyr/posix/unistd.h:219:39: error: 'CONFIG_MAX_TIMER_COUNT' undeclared (first use in this function)
  219 | #define TIMER_MAX                     CONFIG_MAX_TIMER_COUNT
      |                                       ^~~~~~~~~~~~~~~~~~~~~~
/home/yashi/src/zephyr-project/zephyr/lib/posix/options/sysconf.c:257:24: note: in expansion of macro 'TIMER_MAX'
  257 |                 return TIMER_MAX;
      |                        ^~~~~~~~~
[99/110] Building C object zephyr/kernel/CMakeFiles/kernel.dir/sched.c.obj
ninja: build stopped: subcommand failed.
FATAL ERROR: command exited with status 1: /usr/bin/cmake --build /home/yashi/src/zephyr-project/build
```

--- a/include/zephyr/posix/unistd.h
+++ b/include/zephyr/posix/unistd.h
@@ -134,7 +134,8 @@ extern "C" {
 #define _POSIX_LOGIN_NAME_MAX		    (9)
 #define _POSIX_MAX_CANON		    (255)
 #define _POSIX_MAX_INPUT		    (255)
-#define _POSIX_MQ_OPEN_MAX		    CONFIG_MSG_COUNT_MAX
+#define _POSIX_MQ_OPEN_MAX                  Z_SC_VAL_IFDEF(CONFIG_POSIX_MQUEUE, \
+						CONFIG_MSG_COUNT_MAX)
 #define _POSIX_MQ_PRIO_MAX		    (32)
 #define _POSIX_NAME_MAX			    (14)
 #define _POSIX_NGROUPS_MAX		    (8)
@@ -216,7 +217,7 @@ extern "C" {
 #define SIGQUEUE_MAX		      _POSIX_SIGQUEUE_MAX
 #define STREAM_MAX		      _POSIX_STREAM_MAX
 #define SYMLOOP_MAX		      _POSIX_SYMLOOP_MAX
-#define TIMER_MAX		      CONFIG_MAX_TIMER_COUNT
+#define TIMER_MAX                     Z_SC_VAL_IFDEF(CONFIG_TIMER, CONFIG_MAX_TIMER_COUNT)
 #define TTY_NAME_MAX		      _POSIX_TTY_NAME_MAX
 #define TZNAME_MAX		      _POSIX_TZNAME_MAX
 

